packages/angular_devkit/build_angular/src/utils/index-file/inline-fonts_spec.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { InlineFontsProcessor } from './inline-fonts';
+
+describe('InlineFontsProcessor', () => {
+  const content = `
+  <html>
+    <head>
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    </head>
+    <body></body>
+  </html>`;
+
+  it('should inline supported fonts and icons in HTML', async () => {
+    const source = new InlineFontsProcessor({
```

Comment:
`new` here returns a Promise? @_@
Consider adding type annotation to `source`

Reply:
It's not the new that returns a Promise, but rather the `process` method

```ts
const source = new InlineFontsProcessor({
    WOFFSupportNeeded: false,
    minifyInlinedCSS: true,
}).process(content);
```

Would it be simpler to understand if it was not chained?

```ts

const inlineFontsProcessor = new InlineFontsProcessor({
    WOFFSupportNeeded: false,
    minifyInlinedCSS: true,
});

const source = inlineFontsProcessor.process(content);
```